src/utils/serialize_to_json.py

```diff
@@ -0,0 +1,13 @@
+import math
+
+from src.models.BoardGame import BoardGame
+
+
+class SerializeToJson:
+    @staticmethod
+    def serialize_ordered_board_games(board_games: list[BoardGame]):
+        json_ready_board_games = board_games
+        for game in board_games:
+            if game.overallrank == math.inf:
```

Comment:
Leave a comment here explaining why setting to infinity and than changing it from infinity